etc/buildsys/root/format-tree-filter.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+#  Copyright (c) 2018 Tim Niemueller [www.niemueller.de]
+#
+#  This program is free software; you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation; either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU Library General Public License for more details.
+#
+#  Read the full text in the LICENSE.GPL file in the doc directory.
+
```

Comment:
We should make sure that all command line arguments were specified
```suggestion

set -u

```